### PR TITLE
Update downloads.md for Flink 1.10.0

### PR DESCRIPTION
**Pull request by GJL**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -94,137 +139,6 @@ flink_releases:
           url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-json/1.9.2/flink-json-1.9.2.jar
           asc_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-json/1.9.2/flink-json-1.9.2.jar.asc
           sha_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-json/1.9.2/flink-json-1.9.2.jar.sha1
-  -
-      version_short: 1.8
```
Comment (zentol) [inline _config.yml]: we could move this removal into a separate commit

**Comment by GJL** [comment]
Thanks for the review.

File: _config.yml (hunk 1 above)
Comment (GJL) [reply _config.yml]: done
